core/domain/email_subscription_services_test.py

```diff
@@ -22,8 +22,17 @@
 from core.domain import email_subscription_services
 from core.domain import subscription_services
 from core.platform import models
+from core.storage.email import gae_models
```

Comment:
Why do you import this? Just use `user_models` that you import below.

Reply:
Done!